benchmarks/commands/update/deps.ts

```diff
@@ -2,16 +2,16 @@
 export {
   runBenchmarks,
   bench,
-} from "https://deno.land/std@v0.56.0/testing/bench.ts";
-import * as archive from "https://deno.land/std@v0.55.0/archive/tar.ts";
-import * as denoSync from "https://deno.land/std@v0.55.0/async/mod.ts";
-import * as bytes from "https://deno.land/std@v0.55.0/bytes/mod.ts";
-import * as dateTime from "https://deno.land/std@v0.55.0/datetime/mod.ts";
-import * as colors from "https://deno.land/std@v0.55.0/fmt/colors.ts";
-import * as sha512 from "https://deno.land/std@v0.55.0/hash/sha512.ts";
-import * as http from "https://deno.land/std@v0.55.0/http/mod.ts";
-import * as io from "https://deno.land/std@v0.55.0/io/mod.ts";
-import * as log from "https://deno.land/std@v0.55.0/log/mod.ts";
+} from "https://x.nest.land/std@0.56.0/testing/bench.ts";
+import * as archive from "https://x.nest.land/std@0.55.0/archive/tar.ts";
```

Comment:
Wait
Do we have `x.nest.land/std@0.55.0`?
CC @MartonDev

Reply:
No, sorry we only have the latest